lib/src/main/java/org/team100/lib/config/Feedforward100.java

```diff
@@ -47,7 +47,11 @@ public static Feedforward100 makeNeo() {
         return new Feedforward100(0.122, 0.000, 0.100, 0.065, 0.5);
     }
 
-    public static Feedforward100 makeArmPivot() {
+    public static Feedforward100 makeNeo550() {
+        return new Feedforward100(0.32, 0, 0, .07, 0);
```

Comment:
where did the number 0.32 come from?  0.32 volt-sec per revolution means it's only going about 3400 rpm at 12 volts.

Reply:
This was just what I got from testing it for a few minutes, this number is likely not acurate, i'll add a todo.